extensions/resteasy-reactive/quarkus-resteasy-reactive/deployment/src/main/resources/dev-templates/scores.html

```diff
@@ -1,90 +1,216 @@
-     <div class="row">
-      <div class="col-1"></div>
-      <div class="col-1">
-       Consumes
-      </div>
-      <div class="col">
-       {endpoint.produces}
-      </div>
-     </div>
-    {/if}
-    {#for diagnosticEntry in endpoint.diagnostics.entrySet()}
-     <div class="row">
-      <div class="col-1"></div>
-      <div class="col-1">
-       {diagnosticEntry.key}
-      </div>
-     </div>
-     {#for diagnostic in diagnosticEntry.value}
-      <div class="row">
-       <div class="col-1"></div>
-       <div class="col-1"></div>
-       <div class="col">
-        {diagnostic.message}
-       </div>
-       <div class="col-1">
-        <span class="badge badge-{#if diagnostic.score < 50}danger{#else if diagnostic.score < 100}warning{#else}success{/if}">{diagnostic.score}/100</span>
-       </div>
-      </div>
-     {/for}
-    {/for}
-    <div class="row">
-     <div class="col-1"></div>
-      <div class="col-1">
-        Filters
-      </div>
+{#for endpoint in info:endpointScores.endpoints}
+<div class="accordion"  id="accordionExample">
+  <div class="card">
+
+    <div class="card-header" id="headingOne">
+            <button class="btn btn-block btn-lg collapsed" type="button" data-toggle="collapse"
+                    data-target=#{endpoint.httpMethod} aria-expanded="false"
+                    aria-controls={endpoint.httpMethod}>
+              <code>{endpoint.httpMethod} {endpoint.fullPath}</code>
+              <span class="styleBadge badge badge-{#if endpoint.score < 50}danger{#else if endpoint.score < 100}warning{#else}success{/if}">{endpoint.score}/100</span>
+            </button>
     </div>
-    <div class="row">
-      <div class="col-1"></div>
-      <div class="col-1"></div>
-      <div class="col">
-        <span class="badge badge-light" id="collapse-list" data-toggle="collapse" data-target="#filter">
-          <i class="fas fa-greater-than"></i>
-        </span>
-      </div>
+
+    <div id={endpoint.httpMethod} class="collapse" aria-labelledby="headingOne"
+         data-parent="#accordionExample">
+      <div class="card-body">
+
+        <div class="row" style="margin-left:2%;">
+        {#for diagnosticEntry in endpoint.diagnostics.entrySet()}
+          {#for diagnostic in diagnosticEntry.value}
+        <div class="col-4">
+        <div class="row">
+        <div class="gauge" data-score="{diagnostic.score}">
+          <div class="gauge__body">
+            <div class="gauge__fill"></div>
+            <div class="gauge__cover">{diagnostic.score}%</div>
+          </div>
+        </div>
+        </div>
+
+        <div class="row">
+          <div class="key_style">
+          {diagnosticEntry.key}
+          </div>
+        </div>
+
+        {| <script>
+          function setGaugeValue(gauge, value) {
+          if (value < 0 || value > 1) {
+            return;
+          }
+
+          gauge.querySelector(".gauge__fill").style.transform = `rotate(${value / 2}turn)`;
```

Comment:
Well you need to get rid of this, as it will conflict with your end-of-body script